plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/view/ldap/LdapServerBackupForm.js

```diff
@@ -46,7 +46,11 @@ Ext.define('NX.coreui.view.ldap.LdapServerBackupForm', {
         var me = this,
           values = me.callParent(arguments);
 
-        values.backupMirrorEnabled = values.backupMirrorEnabled ? true : false;
+        if (values.backupMirrorEnabled === "on") {
```

Comment:
Even those inputValue/uncheckedValue are strange as in other places we do not set those properties on checkboxes in order to send true/false vs on/off.
Also in Admin/Http Settings we have optional field sets and they are sending true/false without any special handling